README.md

```diff
@@ -191,14 +191,15 @@ Configurations are split in OSGi and Context-Aware.
 |----------------------------------|:-----------------------------|:---------------------------------------------------------------------------------------------------------------------------|:--------:|
 | Search Index                     | index                        | Index defined in SaaS admin                                                                                                |    x     |
 | Search Project Id                | projectId                    | Project identifier defined in SaaS admin                                                                                   |    x     |
-| Search Filters                   | searchFilters                | Key/value pairs of **field name** and **
-value**                                                                            |          |
+| Search Filters                   | searchFilters                | Key/value pairs of **field
+name** and **                                                                                   |          |
+| value**                          |                              |                                                                                                                            |          |
 | Search result highlight tag name | highlightTagName             | The name of the tag that will be used to highlight portions of text in the search results. (Optional; Default value: `em`) |          |
 | Enable Best Bets                 | enableBestBets               | Flag that enables displaying best bets on the top of the search results. Defaults to `false`                               |          |
 | Enable Auto Suggest              | enableAutoSuggest            | Flag that enables auto suggest feature in the search component. Defaults to `true`                                         |          |
-| Enable Autocomplete              | enableAutocomplete           | Flag that enables autocomplete feature in the search input. Defaults to `true`                                             |          |
-| Autocomplete Trigger Threshold   | autocompleteTriggerThreshold | The minimum number of search input characters required for displaying autocomplete options.                                |          |
-| Autocomplete options Max Limit   | autocompleteOptionsMax       | The maximum number of autocomplete options displayed.                                                                      |          |
+| Enable Auto Complete             | enableAutoComplete           | Flag that enables auto-complete feature in the search input. Defaults to `true`                                            |          |
```

Comment:
**Thought**
auto complete / auto-complete? :D 

Is auto-complete used as an adjective?

This English naming can be mental! XD

Reply:
Important is that we are consitent. Let's use "Autocomplete" everywhere as in wikipedia https://en.wikipedia.org/wiki/Autocomplete

Reply:
> Important is that we are consitent. Let's use "Autocomplete" everywhere as in wikipedia https://en.wikipedia.org/wiki/Autocomplete . so no dashes, spaces nor camelcase

oh, so the PR is obsolete then :D
will decline it